core/src/main/java/com/severell/core/mail/MailLogDriver.java

```diff
@@ -33,10 +33,15 @@ public void send() throws ViewException {
         if(text != null) {
             System.out.println(String.format("Plain Text Message: %s", text));
         }
-        String html = getHTML();
 
-        if(html != null) {
-            System.out.println(String.format("HTML Message: %s", getHTML()));
+        if(html != null){
```

Comment:
@Belovedbb Do you think it should be either or? Currently if you set both HTML and Template it will send both in the email. Should it be and if...else if? Whats your opinion?

Reply:
Oh well, it depends, if we allow both, this can enable user to hand craft their html and also pass templated html, though i agree, it seems kind of wasteful. The other option (if else) is kind of restrictive, i'm still yet to determine if that's a good thing or a bad thing. 
On the other hand, we can overload the html method to take in template param and do the conversion to html, while calling the base html method. I think that will keep it clean

Reply:
I think you're right. It will just add another part to the multipart email. It gives the user more control this way.